doc/source/release/1.15.0-notes.rst

```diff
@@ -17,96 +17,1431 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
 Our development attention will now shift to bug-fix releases on the
 1.15.x branch, and on adding new features on the main branch.
 
-This release requires Python 3.10+ and NumPy 1.23.5 or greater.
+This release requires Python 3.10-3.13 and NumPy 1.23.5 or greater.
```

Comment:
Oh and I see that SPEC 0 also recommends dropping support for Python 3.10 in Q4 2024.

Reply:
Re: Python version, it sounds fine to stay in sync with NumPy as long as they don't get too far behind.

Re: Python versions, it will be very helpful when the minimum NumPy version is 2.0+.

Note that we will actually be past the SPEC 0 support window for NumPy 1.25 when SciPy 1.15 is released, but we tend to lag behind by a version or so, perhaps to avoid ending up on the borderline.

If we do NumPy 1.24 for 1.15 (Jan 2025), NumPy 1.25 for SciPy 1.16 (June 2025), and NumPy 1.26 for SciPy 1.17 (Jan 2026), we could probably skip to NumPy 2.1 for SciPy 1.18 (Jun 2026).